### PR TITLE
Add modified at to probes and labels

### LINKED ISSUE #109: Add modified at column to labels and probes tables
This column will be set to the created date at creation and to the current timestamp on any later modificaiton. Primary use for this is migrating new predefined labels and probes (#108 )

### PR DESCRIPTION
**Pull request by rymurr**
closes #109

### CODE REVIEW

**Comment by rymurr** [comment]
depends on #119

**Review by jinfengni — APPROVED** [review]
LGTM

two nit comments.

File: bootstrap/004_labels.sql [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
+BEGIN
+    -- Add modified at column
+    IF (NOT EXISTS(SELECT * FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_SCHEMA = 'INTERNAL' AND TABLE_NAME = 'LABELS' AND COLUMN_NAME = 'LABEL_MODIFIED_AT')) THEN
+        ALTER TABLE IF EXISTS INTERNAL.LABELS ADD COLUMN LABEL_MODIFIED_AT TIMESTAMP;
+        UPDATE INTERNAL.LABELS SET LABEL_MODIFIED_AT = CURRENT_TIMESTAMP WHERE LABEL_MODIFIED_AT IS NULL;
+    END IF;
+
+EXCEPTION
+    WHEN OTHER THEN
+        SYSTEM$LOG('error', 'Failed to migrate probes table. ' || :SQLCODE || ': ' || :SQLERRM);
```
[the remark this answers is not in the document]
Comment (rymurr) [reply bootstrap/004_labels.sql]: fixed

File: bootstrap/004_labels.sql [hunk 2]
```diff
@@ -1,5 +1,21 @@
...
-CREATE TABLE INTERNAL.LABELS if not exists (name string, group_name string null, group_rank number, label_created_at timestamp, condition string, enabled boolean);
+CREATE TABLE INTERNAL.LABELS if not exists (name string, group_name string null, group_rank number, label_created_at timestamp, condition string, enabled boolean, label_modified_at timestamp);
+
+CREATE OR REPLACE PROCEDURE INTERNAL.MIGRATE_LABELS_TABLE()
+RETURNS OBJECT
+AS
+BEGIN
+    -- Add modified at column
+    IF (NOT EXISTS(SELECT * FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_SCHEMA = 'INTERNAL' AND TABLE_NAME = 'LABELS' AND COLUMN_NAME = 'LABEL_MODIFIED_AT')) THEN
+        ALTER TABLE IF EXISTS INTERNAL.LABELS ADD COLUMN LABEL_MODIFIED_AT TIMESTAMP;
```
[the remark this answers is not in the document]
Comment (rymurr) [reply bootstrap/004_labels.sql]: fixed

**Review by jinfengni — APPROVED** [review]
LGTM

**Pull request merged**